proto/winch/config/auth.proto

```diff
@@ -19,6 +19,7 @@ message AuthSource {
         KubernetesAccess kube = 3;
         OIDCAccess oidc = 4;
         TokenAccess token = 5;
+        GoogleServiceAccountOIDCAccess service_account_oidc = 6;
```

Comment:
Can you add more comments for on how do these things work? For example I don't know what is the difference between OIDCAccess and GoogleOIDCAccess.

Reply:
It is described in the proto.. under `GoogleServiceAccountOIDCAccess` and `OIDCAccess`